nservicebus/upgrades/supported-platforms.md

```diff
@@ -13,15 +13,43 @@ NServiceBus is supported on the .NET Framework and .NET Core.
 Note: If possible, packages will target [.NET Standard](https://docs.microsoft.com/en-us/dotnet/standard/net-standard) instead of individual platforms. While this means that packages might work with additional platforms, only the .NET Framework and .NET Core are officially supported by Particular Software.
 
 
-### .NET Framework
+## .NET Framework
 
 NServiceBus is supported for applications targeting the .NET Framework on Windows. For the list of supported Windows versions, refer to [.NET Framework Versions and Dependencies](https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies).
 
 
-### .NET Core
+## .NET Core
 
 NServiceBus is supported for applications targeting .NET Core on a variety of operating systems:
 
 * **Windows**: All Windows versions according to [.NET Core 2.0 - Supported OS versions](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) are fully supported.
 * **Linux**: All Linux versions according to [.NET Core 2.0 - Supported OS versions](https://github.com/dotnet/core/blob/master/release-notes/2.0/2.0-supported-os.md) are fully supported.
 * **macOS**: macOS is supported only for development purposes.
+
+### Packages not supporting .NET Core
+
+Some packages cannot currently support .NET Core or running on non-Windows platforms:
+
+* Transports
+  * NServiceBus.AmazonSQS - Blocked from running on Linux due to a [bug in the AWS SDK](https://github.com/aws/aws-sdk-net/issues/796).
```

Comment:
I think we have a blocking issue about SQS in PlatDev that needs to be resolved first.

Reply:
Don't think this needs to block on a PlatDev issue. This is the state of things as it exists on NuGet right now. When that issue resolves (I'll add a note to the PoA) this should be updated to reflect it.